db/package.js

```diff
@@ -188,13 +188,15 @@ class PackageDbController {
   }
 
   async getOwnedPackages ({ userId, registry, language }) {
-    return this.db.collection('packages').find({
+    const pkgs = await this.db.collection('packages').find({
       'maintainers.userId': userId,
 
       // optional qualifiers
       ...(registry && { registry }),
       ...(language && { language })
     }).toArray()
+
+    return pkgs.map(({ _id: id, ...rest }) => ({ id: id.toString(), ...rest }))
```

Comment:
should we be using "toString()" here?

Reply:
I like the flexibility of leaving off `toString` calls in DB methods; Fastify will auto call `.toString()` on anything the schema dictates as being a string. I don't care much either way though.